etc/kayobe/kolla.yml

```diff
@@ -346,6 +346,11 @@ kolla_build_blocks:
     {% set magnum_capi_packages = ['magnum-capi-helm==1.1.0'] %}
     RUN {{ macros.install_pip(magnum_capi_packages | customizable("pip_packages")) }}
     {% endraw %}
+  nova_base_footer: |
+    {% raw %}
+    {% set nova_packages = ['cryptography==43.0.0'] %}
```

Comment:
I assume cryptography is pulled in as a dependency to Nova? Just bump it up in upper-constraints using a macro?

Reply:
I wasn't sure what the best approach is as adjusting upper-constraints would impact more than just Nova.

Reply:
Well, we use u-c fork as a repo, but there are macros in Kolla that mangle u-c entries - you could use that only for Nova.